cwl/src/main/scala-2.12/wdl4s/cwl/CwlInputParsing.scala

```diff
@@ -1,30 +1,29 @@
 package wdl4s.cwl
 
 import io.circe._
+import io.circe.Decoder
+import io.circe.parser._
 import io.circe.shapes._
 import io.circe.generic.auto._
-import eu.timepit.refined.string._
 import io.circe.refined._
 import io.circe.yaml
 import io.circe.literal._
 import lenthall.validation.Checked._
 import wdl4s.wom.executable.Executable.{InputParsingFunction, ParsedInputMap}
 
 object CwlInputParsing {
-  implicit val fileD = implicitly[Decoder[File]]
-  implicit val stringD = implicitly[Decoder[String]]
-  implicit val intD = implicitly[Decoder[Int]]
-  implicit val longD = implicitly[Decoder[Long]]
-  implicit val floatD = implicitly[Decoder[Float]]
-  implicit val doubleD = implicitly[Decoder[Double]]
-  implicit val booleanD = implicitly[Decoder[Boolean]]
-  implicit val inputArray = implicitly[Decoder[Array[MyriadInputValuePrimitives]]]
+
+  implicit val f = implicitly[Decoder[File]]
```

Comment:
there is some issue with deriving decoders that makes this necessary.  I haven't quite pinpointed it but it seems to involve a member of a `Coproduct` having a Witness field.